index.html

```diff
@@ -0,0 +1,104 @@
+    <!-- Place favicon.ico in the root directory -->
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+
+    <nav>
+      <img class="banner-logo" src="http://i.imgur.com/5tFwpoN.png">
+      <h1> Delicious <br> THE BEST FOOD BLOG ON THE WEB </h1>
+      <ul>
+        <li> <img src="http://i.imgur.com/Xz0logE.png"> </li>
+        <li> <img src="http://i.imgur.com/FnlhJVH.png"> </li>
+        <li> <img src="http://i.imgur.com/pfmEjMI.png"> </li>
+        <li> <img src="http://i.imgur.com/ylgYf41.png"> </li>
+        <li> <img src="http://i.imgur.com/zSxL2IW.png"> </li>
+        <li> <img src="http://i.imgur.com/vCWEjwG.png"> </li>
+        <li> <img src="http://i.imgur.com/wxiFpvM.png"> </li>
+        <li> <img src="http://i.imgur.com/jrfNOkO.png"> </li>
+      <ul>
+    </nav>
+
+    <article>
+      <section class="instructions">
+        <h1> Chocolate Pizza </h1>
+        <p> POSTED ON <time datetime="2013-12-15"> 15 DEC 2013 </time> / DESSERTS </p>
+        <ul>
+          <li> <img src="http://i.imgur.com/mtX28lM.png"> </li>
+          <li> PRINT </li>
+        </ul>
+        <img class="food-pic" src="http://i.imgur.com/fB4svJW.png">
+
+        <p> For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stires, set aside and let cool completely. </p>
+
+        <p> Ice cream: In a small pot over medium head, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled. </p>
+
+        <p> Using an ice cream machine, pour liquids into the frozen basin and process according to the manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving. </p>
+      </section>
+      <section class="ingredients">
+        <img src="http://i.imgur.com/Tbl1pWm.png"
+        <ul>
+          <li> 1 1/2 cups milk </li>
+          <li> 1/2 cup mascarpone </li>
+          <li> 1/2 tsp pink salt </li>
+          <li> 1 lb Black Mission Figs </li>
+          <li> 1/2 cup brown sugar </li>
+          <li> 2-4 tbsp water </li>
+          <li> 1 /12 cups heavy cream </li>
+          <li> 1/3 granulated sugar </li>
+          <li> 2 egg yolks </li>
+          <li> 1 lemon, juiced </li>
+          <li> 2 tbsp butter </li>
+          <li> 1 cupt honey roasted pecans, roughly chopped </li>
+        </ul>
```

Comment:
This is fine for now, but this may change as we go forward to
```
<label><input type="checkbox" name="recipeIngredient" value="chocolate">Chocolate</label>
```
Depending on whether we add JS interactivity or not.